plugins/nf-float/src/main/com/memverge/nextflow/FloatGridExecutor.groovy

```diff
@@ -397,4 +414,14 @@ class FloatGridExecutor extends AbstractGridExecutor {
             'NoAvailableHost'  : QueueStatus.ERROR,
             'Unknown'          : QueueStatus.UNKNOWN,
     ]
+
+    @Override
+    boolean isContainerNative() {
+        return true
+    }
+
+    @Override
+    boolean isFusionEnabled() {
+        return FusionHelper.isFusionEnabled(session)
+    }
```

Comment:
Let's remove this part for now and add the Fusion support in a separate effort. I think more changes will be needed for Fusion so don't want to do it half-way.

Reply:
Agree.  Let's focus on s3 support first.